packages/edit-site/src/store/index.js

```diff
@@ -12,15 +12,24 @@ import * as actions from './actions';
 import * as selectors from './selectors';
 import { STORE_NAME } from './constants';
 
+export let store = null;
+
 export default function registerEditSiteStore( initialState ) {
```

Comment:
It's out of the scope of this PR, but the way this store is initialized is a bit concerning. It could be `null`, but I guess it was also the case before. I'm curious why the initial state has to be set explicitly rather than dispatched with an action.

Reply:
I don't know the reason for that either 🤔 I'll ask around and experiment so we can create a follow-up PR to clean it up.